lang/en.json

```diff
@@ -178,6 +179,12 @@
         "other": "{number} legendary actions"
       }
     },
+    "LongRest": {
+      "Label": "End of a Long Rest"
+    },
+    "ShortRest": {
+      "Label": "End of a Short or Long Rest"
```

Comment:
This makes sense for recovery, but I'm not sure about activations. It wouldn't be very useful to prompt to activate Arcane Recovery at the end of a long rest, for example. Should activations be limited to exactly the rest period they specify? The user could fairly easily add both activation periods if that's what they intend.

Reply:
Makes sense.